code/modules/projectiles/projectile/bullets/pistol.dm

```diff
@@ -55,17 +55,19 @@
 // 12.7mm
 /obj/item/projectile/bullet/a127mm
 	name = "12.7mm FMJ bullet"
-	damage = 55
-	armour_penetration = 0.35
+	damage = 30 //55
```

Comment:
I hope you understand exactly what this is doing. Essentially this will make the 12.7mm pistol weaker on average than a 45 with +P or a 44 pistol with SWC.

What it feels like to me is that this PR is weakening a tier 5 weapon to allow it to be used more frequently. The 12.7 is meant to be dangerously powerful because it's so very difficult to find. I don't think it's a particularly good idea to make it a worse weapon just so people can use it more, because then it becomes no different than other mid range pistols loaded with AP ammo.

Reply:
Also, a hitscan pistol is an absolutely broken buff for a weapon that can be crafted with just a gun book and a 1000 caps or so. Mining isn't that hard.

Reply:
The outlaw recipe needs an actual 12.7 pistol to make. I don't feel like doing much with this anyhow so if it's too op in it's current state then someone can just close it.